src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
+  }
+
+  return result;
```

Comment:
Checklist point 2, use `const`s for constant values. Also checklist point 5. Try to come up with idea how to avoid writing else if.